sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 
 - Prohibit sending the typing notification request within 8 seconds of the previous request.
 
+### Features Added
+
+- Updated to @azure/communication-signaling@1.0.0-beta.12.
```

Comment:
is communication-signaling something that consumers of communication-chat will be aware of? I don't think it's worth mentioning if it's not public surface they have to reason about

Reply:
customers should be able to see azure/communication-signaling as a dependency of communication-chat. The previous changelogs include messages that indicate updates to the signaling package, so I did the same.

Reply:
I have no problem with it